src/kernel/src/budget/mod.rs

```diff
@@ -57,6 +59,21 @@ impl BudgetManager {
     }
 }
 
+#[derive(Debug)]
+pub enum BudgetType {
+    DirectMemory = 1,
+    VirtualMemory = 2,
+    LockedMemory = 3,
+    CpuSet = 4,
+    FdFile = 5,
+    FdSocket = 6,
+    FdEqueue = 7,
+    FdPipe = 8,
+    FdDevice = 9,
+    Threads = 10,
+    FdIpcSocket = 11,
+}
+
```

Comment:
Where did you get this from?

Reply:
<img width="731" alt="image" src="https://github.com/obhq/obliteration/assets/48620541/fd46cf6e-cf65-40d9-89fd-a34f053c0ced">
<img width="565" alt="image" src="https://github.com/obhq/obliteration/assets/48620541/776cf84e-ea40-4eea-bc97-7affe0f548ba">
It's in budget_resource_use

Reply:
Thanks!